src/conn/conn_prefetch.c

```diff
@@ -77,6 +77,9 @@ __wt_prefetch_thread_run(WT_SESSION_IMPL *session, WT_THREAD *thread)
     WT_ASSERT(session, session->id != 0);
     conn = S2C(session);
 
+    /* Mark the session as an eviction thread session. */
```

Comment:
The comment needs to be adjusted to `prefetch`.

Reply:
Yes sorry, just re-pushed.